metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DataStructureFunctions.java

```diff
@@ -138,9 +138,9 @@ public Object apply(List<Object> args) {
   }
 
   @Stellar(name="IS_EMPTY"
-          , description="Returns true if string or collection is empty and false otherwise"
-          , params = { "input - Object of string or collection type (e.g. list)"}
-          , returns = "Boolean"
+          , description="Returns true if string or collection is empty and false if otherwise."
+          , params = { "input - Object of string or collection type (for example, list)"}
+          , returns = "True if the string or collection is empty and false otherwise."
```

Comment:
This conflicts with work from https://github.com/apache/incubator-metron/pull/324